src/components/MostRecentScoreboard.tsx

```diff
@@ -28,20 +52,45 @@ function MostRecentScoreboard(props: MostRecentScoresProps) {
   };
 
   return (
-    <div className={styles.grid}>
-      <InfiniteScroll
-        dataLength={items.length} //This is important field to render the next data
-        next={fetchData}
-        hasMore={items.length < plays.length}
-        loader={<LoadingCircle />}
-        endMessage={
-          <p style={{ textAlign: "center" }}>
-            <b>Yay! You have seen it all</b>
-          </p>
-        }
+    <div>
+      <Tabs
+        value={tabIndex}
+        onChange={handleTabClick}
+        centered
+        sx={{
+          maxWidth: "500px",
+        }}
       >
-        {items}
-      </InfiniteScroll>
+        {sports.map((sport, index) => (
+          <Tab
+            icon={<Avatar src={`${sport}.jpeg`} variant="square" />}
+            key={sport}
+            sx={{
+              backgroundColor: tabIndex === index ? "currentColor" : "#676b72",
```

Comment:
`"#676b72"` should be replaced by `.App-container { background-color:` in `App.css` but I am too dumb to know how to wire them together.